README.md

```diff
@@ -216,6 +217,7 @@ This model represents information about arguments provided to a specific `potent
 
 ## Dependencies
 
+* Rails - Versions 5.2, 6, and 6.1 are supported.
```

Comment:
I think this will actually work without Rails, but I haven't actually tested that.